src/plasma/test/manager_tests.cc

```diff
@@ -258,31 +258,38 @@ TEST object_notifications_test(void) {
   CHECK(fcntl(fd[1], F_SETFL, flags | O_NONBLOCK) == 0);
 
   ObjectID oid = globally_unique_id();
```

Comment:
I realize you didn't introduce it in this PR, but can you change `oid` to `object_id`?